lib/rdf/vocab.rb

```diff
@@ -249,6 +249,11 @@ module Vocab
       ore:    {uri: "http://www.openarchives.org/ore/terms/"},
       org:    {uri: "http://www.w3.org/ns/org#"},
       owl:    {uri: "http://www.w3.org/2002/07/owl#", alias: true},
+      pcdm_models: {
```

Comment:
maybe just `pcdm:`?

Reply:
You are right, thanks for catching this error.